src/components/Badge/Badge.stories.mdx

```diff
@@ -0,0 +1,255 @@
+import { useState } from 'react';
+import { action } from '@storybook/addon-actions';
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs/blocks';
+import Badge from './Badge';
+import Box from '../Box/Box';
+import Button from '../Button/Button';
+
+<Meta
+  title="Components/Badge"
+  component={Badge}
+/>
+
+# Badge
+
+Use badge to display feedback for users about specific actions, or states of an application. All that is required to render a basic version of an Badge is a `message` prop which will contain text
```

Comment:
Description needs updating. Badge description shouldn't be the same as an Alert. They are used to communicate status.

Reply:
This still needs updating